### ISSUE TITLE
No orders being emitted by socket

### ISSUE THREAD

**Issue by dj2ball** [etherdelta/etherdelta.github.io#287]
I've been having an intermittent issue over the last few days where randomly the socket will stop emitting new order data....is anyone else seeing this behaviour? The socket connection is stable and  I get a continuous stream of funds/trade events but nothing coming through with new orders....?

Any ideas?

**Comment by chenshak** [comment]
Yes I see this behavior as well. Also, sometimes after connection I won't get any orders at all no matter how long I wait (then I need to disconnect and reconnect multiple times until it works).

In addition after emitting getMarket, the response I get is very inconsistent (using the same input):
sometimes {}
sometimes something like: {"returnTicker": "" , "trades": [], "orders" {"buys": [], "sells": [] } }
sometimes only returnTicker data
sometimes all data including token orders and user funds

**Comment by MeoMix** [comment]
@zackcoburn I am seeing this issue as well. Maybe you tweaked something recently?

**Comment by dj2ball** [comment]
Agreed, for the last 4-5 days, I would say there is a 50/50 shot of connecting to the socket and it just refusing to emit orders. Other events still come through though (trades, funds etc). Like Meomix, I observe this behaviour both by listening for on.(orders) but also see the same result listening to on.(message). And often I will have a perfectly working connectiong disconnect and when it reconnects, it won't pickup orders again until the next connect/reconnect cycle. Also happy to provide logs if it helps?

Currently I'm having to workaround this by having the script check for the last order event and then manually execute a reconnect once they stop coming though - as I'm sure you'll agree this is far from a perfect solution, so any ideas you have would be great.

**Comment by zackcoburn** [comment]
Hmm, I'll have to look into this.

**Comment by djskinner** [comment]
Also seeing this FYI
